Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubectl/resource_printer.go

```diff
@@ -1430,9 +1431,14 @@ func printHorizontalPodAutoscalerList(list *extensions.HorizontalPodAutoscalerLi
 }
 
 func printSecurityContextConstraints(item *api.SecurityContextConstraints, w io.Writer, withNamespace, wide, showAll bool, columnLabels []string) error {
-	_, err := fmt.Fprintf(w, "%s\t%t\t%v\t%t\t%s\t%s\t%s\t%s\n", item.Name, item.AllowPrivilegedContainer,
+	priority := ""
```

Comment:
looks like "<none>" is the default.  There is no describer for SCCs yet, issue is captured in https://github.com/openshift/origin/issues/5299